src/chapters/chapter1.tex

```diff
@@ -10,14 +10,14 @@ \section{Mathematical Notation}
 	you rely on definitions to get yourself out of philosophical jams.  For instance,
 	with our definition of set, consider ``the set of all sets that don't
 	contain themselves.''  Such a set cannot exist!
-	This is called \emph{Russel's Paradox}, and shows
+	This is called \emph{Russell's Paradox}, and shows
 	that if we start talking about sets of sets, we may need more than
 	intuition.}. We write a set with curly-braces $\{$ and $\}$ and
-	list the objects inside.  For instance
+	list the objects inside.  For instance,
 	\[
-		\Set{1,2,3}.
+		\Set{1,2,3}
 	\]
-	This would be read aloud as ``the set containing the elements $1$, $2$, and $3$.''
+	would be read aloud as ``the set containing the elements $1$, $2$, and $3$.''
```

Comment:
You've made the meaning inconsistent...  The preceding sentence describes how to write a set in curly braces.  Then it says "For instance" and then goes on to explain pronunciation. 

Perhaps _For instance, ..., **which** would be read aloud as ...._